README.md

```diff
@@ -34,7 +38,7 @@ The rest of steps assume the environment variable `CUDF_HOME` points to the
 root directory of your clone of the cuDF repo, and that the `cudf_dev` Anaconda
 environment created in step 3 is active.
 
-## Clone, build and install cuSpatial
+### Clone, build and install cuSpatial
 
 1. export `CUSPATIAL_HOME=$(pwd)/cuspatial`
```

Comment:
These instructions are not complete either, since they don't cover building the Cython...

Reply:
true :)